### ISSUE TITLE
Duplicates have to be removed.

### ISSUE THREAD

**Issue by CarlPoirier** [KiCad/SMD_Packages.pretty#20]
Many footprints are now included in more specialized packages, especially after the work of @michal777. The duplicates have to be removed from here. Eventually, this library will completely disappear.

**Comment by pointhi** [comment]
@SchrodingersGat : what about checking if all footprints are present in other libraries, declaring this library as Legacy?

**Comment by CarlPoirier** [comment]
It's read-only, at least. I did one pass back then, I'm not sure there are lots of duplicates left. It may be a matter of finding new homes for the leftover ones.

**Comment by poeschlr** [comment]
From looking at the footprint names, it seems that there are a lot of duplicates.
- The BGA footprints are duplicates of the once in the Housings_BGA lib
- The PLCC footprints are duplicates of the once in Housings_LCC
- The SOIC footprints duplicate the Housings_SOIC footprints
- The SSOP footprints are also in the Housings_SSOP lib

But a few footprints are only found in this lib.
- To SO footprints are unique in the lib, (Should go to the Housings_SOIC lib)
- The PQFP and VQFP footprints of this lib are missing in the Housings_QFP lib
- SOJ should either go into Housings_SOIC or they should get their own lib

The conn footprints are useless without the information for what component they are for. (Some look similar to molex picoblade. Which we have in Connectors_Molex lib.)

**Comment by evanshultz** [comment]
Thanks for dispositioning those footprints. There are a few others not captured above:
- 1Pin: Wire_Pads
- ~~BGA*: All are in Hougs_BGA already~~
- ~~Ceramic_Filter: No description in footprint to help identify the part~~
- ~~Conn*: No description but looks similar to JST SUR; why keep it if users can't match it to a part?~~
- ~~DIP-8_SMD: Remove as we already have a script KLC-compliant one~~
- ~~EMI_Filter: No description but guessing it's some kind of vertical daughterboard with filtering~~
- ~~Fuse_SMD: No description but similar Fuse_Holders_and_Fuses:Fuse_SMD_Schurter_OMT125~~
- LFCSP-40-1EP: Housings_CSP (appears to fit [AD8195](http://www.analog.com/media/en/technical-documentation/data-sheets/AD8195.pdf))
- ~~MLF-44_Atmel: For [Atmel 44M1](http://www.atmel.com/images/doc2502.pdf) but Housings_QFN_DFN:QFN-44-1EP_7x7mm_Pitch0.5mm is better~~
- ~~PLCC: Already have then in Housings_LCC~~
- ~~POT_SMD: No description but Potentiometers:Potentiometer_Trimmer-EVM3E is similar and better~~
- PQFP* (5 footprints): Housings_QFP
- ~~QFN-52-1EP_HandSoldering: We have a better 7x8mm QFN-52~~
- ~~SMD*: We have better resistor, capacitor, inductor, etc. footprints already~~
- ~~SO/SOJ: The fit alongside footprints in Housings_SOIC, but I think these are covered already~~
- SO-8_PowerPAK_Vishay_Single: Housings_SOIC; I feel like I've seen this elsewhere but can't find it now
- ~~SSOP*: We have better TSSOP footprints~~
- SUPSOJ40: Housings_DIP; looks kinda like [this](http://www.topline.tv/drawings/pdf/SOJ/SOXJ40.pdf)
- TSOP32SP: Housings_SSOP; I don't find a datasheet online now but I find hits for TSOP-32 8x16mm
- VLM0806*: TO_SOT_Housings_SMD, I suppose; I finally found it's a typo of [VML0806](http://rohmfs.rohm.com/en/products/databook/package/spec/discrete/vml0806_tr-e.pdf)
- VQFP-240: Move to Housings_QFP; I don't see any VQFP, but I find [other QFPs](https://www.altera.com/content/dam/altera-www/global/en_US/pdfs/literature/ds/pkgds.pdf) that match
- powermite3: Diodes_SMD will hold this [common footprint](https://www.microsemi.com/document-portal/doc_view/11086-sd69a-pdf) we don't yet have

@pointhi @CarlPoirier @poeschlr @SchrodingersGat 
I've put a strike through the parts that I think have no value. Agree?

For the others, they look like they could fill some gaps in our library (with some work to meet KLC, be renamed, etc.). I'll get the 13 footprints above updated and create PRs over the next few weeks unless someone objects to them being added.

**Comment by evanshultz** [comment]
I found the parts that look like SO-8_PowerPAK_Vishay_Single: they're in TO_SOT_Housings_SMD. However, none of those packages have the same geometry so we will want to keep this one.

In addition, these packages (`PQFN_8x8`, `SOT-669_LFPAK`, and `TDSON-8-1*`) all look more like SOIC-8 than TO/SOT. SOT-669 at least has "SOT" in the name, so I guess it could stay. But should the others move to Housings_SOIC along with the Vishay package?

**Comment by evanshultz** [comment]
Status update:
- `1Pin`: Upon further thinking I believe we make a range like `1Pin_4.064mm_24AWG`; thoughts?
- `LFCSP-40-1EP`: I see only 48pin 7x7mm LFCSPs after searching online; can anyone find a suitable part?
- `PowerPAK_SO-8_Single` and `Dual`: Finished, but where should this go (see above)?
- `VML0806`: Finished, but where should this go (perhaps TO_SOT_Housings_SMD)?
- `Powermite*`: Waiting on dimensional info from Microsemi

![image](https://user-images.githubusercontent.com/1936989/29190544-5eb2a144-7dcf-11e7-8b56-991bf6c9b820.png)

**Comment by evanshultz** [comment]
@jkriege2 @poeschlr @Shackmeister 
Can someone please suggest where the VML0806 footprint should go? It's ready to go, but I'm not sure where I should submit the PR. It's like some footprints in TO_SOT_Housings_SMD, but maybe there's a better place?

**Comment by jkriege2** [comment]
I would move the VML0806 there: https://github.com/KiCad/Housings_SON.pretty

JAN

**Comment by evanshultz** [comment]
The 40 lead LFCSP footprint in this repo is 7x7mm.  All 40 pin LFCSP I find are 6x6mm and all 48 lead LFCSP are 7x7mm. I assume there is a 7x7mm 40 lead or this footprint would not be here, but there's no documentation in the footprint and I can't find it online. Can we drop this footprint? Supporting docs:
- http://www.analog.com/en/design-center/packaging-quality-symbols-footprints/package-index/csp-chip-scale-package/lfcsp.html
- https://www.onsemi.com/pub/Collateral/932AC.PDF

Similarly, I don't find a 240 pin VQFP datasheet online. Lots of other pin counts of VQFP, yes, but not 240 leads. Again, if there's not an easily-accessible datasheet is this package important?

If someone can point me to a datasheet I'll happily update these footprints. Otherwise, does anyone object to dropping these footprints when this repo is deprecated?

**Comment by evanshultz** [comment]
@poeschlr @jkriege2 @SchrodingersGat 
Can you please read the comments above and see if this lib can be deprecated? I think it may be good. If there is more left to do, please let me know what to do.

**Comment by SchrodingersGat** [comment]
The PowerPak footprints should go in TO_SOT_SMD library. Any other ones that you are still looking for somewhere to put them?

**Comment by evanshultz** [comment]
PowerPAK were already put into Housings_SOIC. See https://github.com/KiCad/Housings_SOIC.pretty/pull/30. Do you mean they need to be moved?

No. I think it's good and I'm soliciting others to see if they find anything left to do.

**Comment by SchrodingersGat** [comment]
SOIC is fine for those. If no one else raises objections I'm happy to mark this as deprecated and move forward.

**Comment by evanshultz** [comment]
@SchrodingersGat 
Sounds good!

@CarlPoirier @pointhi @poeschlr @jkriege2 
Any comments? I'll leave this Issue open for another day to gather any additional feedback.

**Comment by jkriege2** [comment]
Only a comment on `1pin`... I think that can be savely deleted with the lib of pins/testpoints that I added within the last two weeks ... 
JAN

**Comment by SchrodingersGat** [comment]
I have now removed SMD_Packages.pretty from the lib-tables
